system/greenhouse-ccloud/templates/team-rolebindings.yaml

```
@@ -1,19 +1,18 @@
 {{ $chartName := .Chart.Name }}
 {{ range $roleBinding := .Values.teamRoleBindings }}
-{{- if and $roleBinding.teamRef $roleBinding.teamRoleRef $roleBinding.clusterNames }}
-{{- range $clusterName := $roleBinding.clusterNames }}
+{{- if and $roleBinding.teamRef $roleBinding.teamRoleRef $roleBinding.clusterSelector }}
 ---
 apiVersion: greenhouse.sap/v1alpha1
 kind: TeamRoleBinding
 metadata:
-  name: {{ printf "%s-%s-%s" $roleBinding.teamRef $roleBinding.teamRoleRef $clusterName | trunc 63 | trimSuffix "-" }}
+  name: {{ printf "%s-%s-%s" $roleBinding.teamRef $roleBinding.teamRoleRef | trunc 63 | trimSuffix "-" }}
```

the 'printf' expects 3 values here, but this change removes the third
```suggestion
  name: {{ printf "%s-%s" $roleBinding.teamRef $roleBinding.teamRoleRef | trunc 63 | trimSuffix "-" }}
```